lib/improver/uv_index.py

```diff
@@ -93,5 +93,7 @@ def calculate_uv_index(uv_upward, uv_downward, scale_factor=3.6):
         uv_index.data = (uv_upward.data +
                          uv_downward.data) * scale_factor
         uv_index.standard_name = "ultraviolet_index"
+        uv_index.long_name = "ultraviolet_index"
```

Comment:
It might be worth updating or adding a unit test that checks the `standard_name`, `long_name` and `var_name`, just to avoid any future confusion.

Reply:
I've updated the metadata unit test, so that should cover this now.